core/lib/pipeline/schema-validation.ts

```diff
@@ -8,15 +8,23 @@ import * as SchemaValidator from "z-schema";
 import { OperationAbortedException } from "../exception";
 import { Channel } from "../message";
 import { createPerFilePlugin, PipelinePlugin } from "./common";
+import * as path from "path";
+import { AppRoot } from "../constants";
+
+// TODO-TIM: Find a better way? Move schema to a package?
+const schemaFolder = process.env.JEST_WORKER_ID === undefined ? "." : path.resolve(AppRoot, "../schema");
```

Comment:
Moving the schemas to a package is an interesting idea, for sure.

Reply:
yeah wondering if we can have a non published package with rush which just get statically linked

Reply:
maybe just with the bundledDependencies it would work